domains/buiphuan.json

```
@@ -0,0 +1,10 @@
+{
+  "description": "buiphuan.is-a.dev : This is my personal notebook",
+  "owner": {
+    "username": "anbui0211",
+    "email": "anbui0211@gmail.com"
+  },
+  "record": {
+    "URL": "anbui0211.github.io"
```

website doesnt really contain anything 
also, URL records should contain protocol
```suggestion
    "URL": "https://anbui0211.github.io/"
```

---

@21Z i changed it. Thanks

---

> I haven't started coding. I just got the idea to create a website.

At least try to put something in your website, like an about me for example.